bin/osg-topology

```diff
@@ -131,14 +131,30 @@ def mangle_url(url, args, session = None):
     return urlparse.urlunsplit(url_tuple)
 
 
-def list_security_contacts(args):
+def get_contact_list_info(contact_list):
     """
-    List security contacts for OSG.
+    Get contact list info out of contact list
     """
-    base_url = "https://my.opensciencegrid.org/rgsummary/xml" \
-               "?summary_attrs_showcontact=on&facility=on&gridtype=on" \
-               "&gridtype_1=on&all_resources=on&active_value=1&active=on&" \
-               "disable=on&disable_value=0"
+    contact_list_info = []
+    for contact in contact_list:
+        if contact.tag == 'ContactType' or contact.tag == 'Type':
+            contact_list_type = contact.text.lower()
+        if contact.tag == 'Contacts':
+            for contact in contact:
+                contact_info = { 'ContactType' : contact_list_type }
+                for contact_contents in contact:
+                    contact_info[contact_contents.tag] = contact_contents.text
+                contact_list_info.append(contact_info)
+
+    return contact_list_info
+
+
+def get_contacts(args, urltype, roottype):
+    """
+    Get one type of contacts for OSG.
+    """
+    base_url = "https://my.opensciencegrid.org/" + urltype + "summary/xml?" \
+               "&active=on&active_value=1&disable=on&disable_value=0"
```

Comment:
It looks like we lost `gridtype=on&gridtype_1=on` for resources, waiting on an answer from @matyasselmeci to see if this is relevant.

Reply:
I see what those extra parameters do is to exclude entries for test resources.  Is that important?

Reply:
The [ticket code](https://github.com/opensciencegrid/ticket/blob/master/app/models/Address.php#L6) which sent security email does not exclude the test resources, so I don't think osg-topology should either.

Reply:
AFAIK filtering by gridtype has no meaning for contacts, only for resources and resource downtimes.

Reply:
Well we're filtering on resources then grabbing the contacts from the resultant XML. Including the testing resources seems fine to me, especially if it was the old behavior, so I'll merge this.